e2e/drift/drift_test.go

```diff
@@ -184,6 +184,16 @@ var _ = Describe("Drift", Ordered, func() {
 					`{"name":"http","port":1234,"protocol":"TCP","targetPort":"http-web-svc"}]}}`))
 			})
 		})
+
+		Context("Resource manifests containing status fields", func() {
+			It("Is marked as not ready", func() {
+				bundleName := "drift-correction-test-drift-ignore-status"
+				Eventually(func() bool {
+					b := getBundle(bundleName, k)
+					return b.Status.Summary.Ready == 1
+				}).Should(BeTrue())
+			})
+		})
```

Comment:
I tested this with an older version, and checked that it fails. The bundle never reaches a ready status:
```
❯ kubectl get bundle -n fleet-local -w drift-correction-test-drift-ignore-status
NAME                                        BUNDLEDEPLOYMENTS-READY   STATUS
drift-correction-test-drift-ignore-status   0/1                       Modified(1) [Cluster fleet-local/local]; deployment.apps drift-ignore-status/nginx-deployment modified {"status":{"collisionCount":0}}
```